Assets/Javascript/index2.js

```diff
@@ -0,0 +1,251 @@
+const workSection = document.getElementById('sec_3');
+const mainUl = document.createElement('ul');
+const workDetail = document.getElementById('detail_sec');
+const sectObj = [];
+
+for (let i = 1; i < 6; i += 1) {
+  sectObj.push(document.getElementById(`sec_${i}`));
+}
+
+const workInformation = [{
+  imagePath: './Assets/Images/portfolio_pictures/sn1.png',
+  title: 'Tonic',
+  features: ['CANOPY', 'Back End Dev', '2015'],
+  description: 'A daily selection of privately perzonalized reads; no account or sign-ups required.',
+  languages: ['html', 'css', 'javascript'],
+  buttonText: 'See Project',
+  detailText: " Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent ",
+},
+{
+  imagePath: './Assets/Images/portfolio_pictures/sn2.png',
+  title: 'Multi-Post Stories',
+  features: ['CANOPY', 'Back End Dev', '2015'],
+  description: 'A daily selection of privately perzonalized reads; no account or sign-ups required.',
+  languages: ['html', 'css', 'javascript'],
+  buttonText: 'See Project',
```

Comment:
You're having a code repetition here: this button seems to be present in all projects. I personally think you should find a way to refactor your code so as to follow the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) rule. This will ensure that this button is re-used in all instances where it is needed :blush: